### PR TITLE
feat: migrate from pop-up-api to popover-api

### PR DESCRIPTION
**Pull request by jh3y**
Changes proposed in this pull request:

- To align with the Open UI complete name change of `popup` and `pop-up` to `popover`: https://github.com/openui/open-ui/pull/629
- There will soon be another change for the `defaultopen` attribute/property pages too

### CODE REVIEW

File: site/en/blog/pop-ups-theyre-making-a-resurgence/index.md [hunk 1]
```diff
@@ -10,26 +10,26 @@ tags:
...
-This [change request](https://chromium-review.googlesource.com/c/chromium/src/+/3991667) was merged into Chromium on [November 3, 2022](https://chromium-review.googlesource.com/c/chromium/src/+/3991667). All of the demos and references have been updated to make use of the new `popover` naming convention.
+The Open UI initiative resolved to change the attribute from `popup` to `popover` for the Popover API on [October 27, 2022](https://github.com/openui/open-ui/issues/627).
+This [change request](https://chromium-review.googlesource.com/c/chromium/src/+/3991667) was merged into Chromium on [November 3, 2022](https://chromium-review.googlesource.com/c/chromium/src/+/3991667). References have been updated to make use of the new `popover` naming convention.
 {% endAside %}
 
 The goal of the [Open UI initiative](https://open-ui.org/) is to make it easier for developers to make great user experiences. To do this, we are trying to tackle the more problematic patterns that developers face. We can do this by providing better platform built-in APIs and components.
 
-One such problem area is pop-ups.
+One such problem area is pop-ups. Or as they are referred to in Open UI, "Popovers".
```
Comment (jh3y) [inline site/en/blog/pop-ups-theyre-making-a-resurgence/index.md:23]: @rachelandrew – Do you perhaps have a better way to word this? Basically, the Open UI has decided to rename the API and references towards it. But, people are naturally going to refer to this still as pop-ups for some part. I want to somehow sum that up in a line here.

**Comment by jh3y** [comment]
The changes have now gone into the Open UI explainer: https://github.com/openui/open-ui/pull/629

File: site/en/blog/pop-ups-theyre-making-a-resurgence/index.md (hunk 1 above)
Comment (rachelandrew) [inline site/en/blog/pop-ups-theyre-making-a-resurgence/index.md:23]:
```suggestion
One such problem area is pop-ups, described in Open UI as "Popovers".
```

**Review by rachelandrew — APPROVED** [review]
I just made a suggestion for wording.

**Pull request merged**